manifests/init.pp

```diff
@@ -35,6 +38,7 @@
   Boolean $manage_repo,
   Boolean $oss,
   Optional[String] $package_source,
+  Optional[Stdlib::Absolutepath] $plugindir,
```

Comment:
Please set the default value here (the line should then be moved after lines without default value):

```suggestion
  Optional[Stdlib::Absolutepath] $plugindir = undef,
```

This allows puppet-string to get the default value when generating the documentation.